### ISSUE TITLE
Client-setup documentation refers to "App" when it should be "app" (Lowercase)

### ISSUE THREAD

**Issue by juanparati** [inertiajs/inertia#921]
### Versions:

- `@inertiajs/inertia` version: 0.10.1
- `@inertiajs/inertia-vue` version: 0.7.2
-  `Typescript` version: 3.9.5

### Describe the problem:

The [client setup documentation](https://inertiajs.com/client-side-setup), it refers to parameter "App" but it should be "app" because in other ways the following error is raised by Typescript transpiler:

`TS2339: Property 'App' does not exist on type '{ el: Element; app: InertiaApp; props: { attrs: { id: string; 'data-page': string; }; props: InertiaProps; }; }'.`

### Steps to reproduce:

1. Setup inertiajs and inertiajs-vue with Typescript according to the [client setup documentation](https://inertiajs.com/client-side-setup).
2. run: `npm run dev`

**Comment by claudiodekker** [comment]
Hi @juanparati,

We've actually deprecated the lowercase 'App' type. The problem is that our TypeScript definitions just aren't reporting this properly. Once the TS-conversion of the Vue adapters is done, this problem should be solved.

Thanks!

**Comment by reinink** [comment]
Hey! Thanks so much for your interest in Inertia.js and for sharing this issue/suggestion.

In an attempt to get on top of the issues and pull requests on this project I am going through all the older issues and PRs and closing them, as there's a decent chance that they have since been resolved or are simply not relevant any longer. My hope is that with a "clean slate" me and the other project maintainers will be able to better keep on top of issues and PRs moving forward.

Of course there's a chance that this issue is still relevant, and if that's the case feel free to simply submit a new issue. The only thing I ask is that you please include a super minimal reproduction of the issue as a Git repo. This makes it much easier for us to reproduce things on our end and ultimately fix it.

Really not trying to be dismissive here, I just need to find a way to get this project back into a state that I am able to maintain it. Hope that makes sense! ❤️
